hedera-node/hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/usage/crypto/CryptoOpsUsage.java

```diff
@@ -192,13 +192,13 @@ public void cryptoCreateUsage(
 
         if (maxAutomaticTokenAssociations > 0) {
             baseSize += INT_SIZE;
+            accumulator.addRbs(maxAutomaticTokenAssociations * lifeTime * CREATE_SLOT_MULTIPLIER);
         }
 
         /* Variable bytes plus two additional longs for balance and auto-renew period;
         plus a boolean for receiver sig required. */
         accumulator.addBpt(baseSize + 2 * LONG_SIZE + BOOL_SIZE);
         accumulator.addRbs((CRYPTO_ENTITY_SIZES.fixedBytesInAccountRepr() + baseSize) * lifeTime);
-        accumulator.addRbs(maxAutomaticTokenAssociations * lifeTime * CREATE_SLOT_MULTIPLIER);
```

Comment:
is this change needed? If the maxAutomaticTokenAssociations is 0, it would just call accumulator.addRbs(0) anyways